.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '30 * * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v3
```

Comment:
Just fyi looks like there is a `v4`, now.

Reply:
Thanks, will check.

Reply:
```suggestion
    - uses: actions/stale@v4
```